tests/chart_tests/test_logging_sidecar.py

```diff
@@ -42,3 +44,50 @@ def test_logging_sidecar_config_disabled(self, kube_version):
             show_only="templates/logging-sidecar-configmap.yaml",
         )
         assert len(docs) == 0
+
+    def test_logging_sidecar_custom_config(self, kube_version):
+        """Test logging sidecar config with customConfig flag enabled"""
+        test_custom_sidecar_config = textwrap.dedent(
+            """
+        loggingSidecar:
+          enabled: true
+          customConfig: true
+          name: sidecar-logging-consumer
+          airflowSidecarConfig: |
+              log_schema:
+                timestamp_key : "@timestamp"
+              data_dir: "${SIDECAR_LOGS}"
+              sources:
+                generate_syslog:
+                  type: file
+                  include:
+                    - "${SIDECAR_LOGS}/*.log"
+                  read_from: beginning
+              transforms:
+                transform_syslog:
+                  type: add_fields
+                  inputs:
+                    - generate_syslog
+                  fields:
+                    component: "${COMPONENT:--}"
+                    workspace: "${WORKSPACE:--}"
+                    release: "${RELEASE:--}"
+              sinks:
+                out:
+                  type: datadog
+                  inputs:
+                    - transform_syslog
```

Comment:
The indentation in this block is inconsistent and fails yamllint.
```suggestion
        loggingSidecar:
          enabled: true
          customConfig: true
          name: sidecar-logging-consumer
          airflowSidecarConfig: |
            log_schema:
              timestamp_key : "@timestamp"
            data_dir: "${SIDECAR_LOGS}"
            sources:
              generate_syslog:
                type: file
                include:
                  - "${SIDECAR_LOGS}/*.log"
                read_from: beginning
            transforms:
              transform_syslog:
                type: add_fields
                inputs:
                  - generate_syslog
                fields:
                  component: "${COMPONENT:--}"
                  workspace: "${WORKSPACE:--}"
                  release: "${RELEASE:--}"
            sinks:
              out:
                type: datadog
                inputs:
                  - transform_syslog
```

Reply:
done